### DISCUSSION TITLE
Get a subset

### DISCUSSION THREAD

**Discussion by powersoft** [skyfielders/python-skyfield#939] [General]
I have calculated for stars the sets,

alt, azi, distance = astrometric.apparent().altaz()

But now I wont to make a subset  alt', azi',distance' for all alt>0

Have no clou how to do that, please need some help

**Comment by brandon-rhodes** [comment]
Try NumPy's ability to index by conditionals — you can use something like `alt.degrees > 0.0` as an index:

https://www.codecademy.com/learn/introduction-to-numpy-for-finance/modules/learn-numpy-introduction-for-finance/cheatsheet
